lib.php

```diff
@@ -132,8 +132,10 @@ function oublog_delete_instance($oublogid) {
     }
 
     if ($oublog->global) {
-        throw new moodle_exception('deleteglobalblog', 'oublog');
+        debugging("Skipping deletion of global blog (ID: {$oublogid}). Global blogs cannot be deleted.", DEBUG_DEVELOPER);
+        return false; // Prevent cron failure, but do not delete the blog
```

Comment:
inline comments should end with proper punctuation marks
https://moodledev.io/general/development/policies/codingstyle#inline-comments